simtools/utils/names.py

```diff
@@ -19,59 +17,53 @@
     "simtel_array_config_file_name",
     "simtel_single_mirror_list_file_name",
     "simtel_telescope_config_file_name",
-    "get_telescope_name_db",
-    "split_telescope_model_name",
-    "telescope_model_name_from_array_element_id",
-    "translate_simtools_to_corsika",
     "validate_array_layout_name",
     "validate_model_version_name",
     "validate_site_name",
-    "validate_sub_system_name",
     "validate_telescope_id_name",
-    "validate_telescope_model_name",
-    "validate_telescope_name_db",
+    "validate_telescope_name",
 ]
 
-lst = "LST"
-mst = "MST"
-sct = "SCT"
-sst = "SST"
-hess = "HESS"
-magic = "MAGIC"
-veritas = "VERITAS"
-
-all_telescope_class_names = {
-    lst: ["lst"],
-    mst: ["mst"],
-    sct: ["sct"],
-    sst: ["sst"],
-    hess: ["hess"],
-    magic: ["magic"],
-    veritas: ["veritas"],
+# Telescopes and other array elements
+array_element_names = {
+    # CTAO telescopes
+    "LSTN": {"site": "North", "observatory": "CTAO", "class": "telescope"},
+    "MSTN": {"site": "North", "observatory": "CTAO", "class": "telescope"},
+    "LSTS": {"site": "South", "observatory": "CTAO", "class": "telescope"},
+    "MSTS": {"site": "South", "observatory": "CTAO", "class": "telescope"},
+    "SSTS": {"site": "South", "observatory": "CTAO", "class": "telescope"},
+    "SCTS": {"site": "South", "observatory": "CTAO", "class": "telescope"},
+    # calibration devices
+    "ILLN": {"site": "North", "observatory": "CTAO", "class": "calibration"},
```

Comment:
You mean what `ILLN` North means? I think I remember.

I anyway plan to remove all those lines and move them into the json schema (some parts are already duplicated, e.g., the list of telescopes) - names.py will read then the schema files to get the naming dicts.

Reply:
Yes, ILLN, RLDN, STPN, etc. I don't remember what all of them means and I think it would be useful to have the non-abbreviated names as well. Also if it's in the schema files it would be useful.